css/styles.css

```diff
@@ -0,0 +1,91 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Crimson+Pro:wght@200..900&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  text-align: center;
+  align-items: center;
+  background-color: #1c1a19;
+  background-image: url("/myPortfolio/Illustration.png");
```

Comment:
- [ ] Kindly fix the link to your background image so that it displays correctly. Consider using `../Illustration.png`. I will suggest you move the background image property to your headline section because from the template, the background is only for the headline section.

Reply:
Done. Thanks.